ui/app/AppLayouts/Chat/views/communities/CommunityNewTokenView.qml

```diff
@@ -225,6 +225,11 @@ StatusScrollView {
                         expression: { return getColor(model.colorId) }
                     }
                 ]
+                filters: ValueFilter {
+                        roleName: "walletType"
+                        value: Constants.watchWalletType
+                        inverted: true
```

Comment:
What about keycard? (aka `Constants.keyWalletType`), thouse should imo be excluded too

Reply:
Why? IDK if there's any difference between a wallet account imported from seed phrase or keycard from this perspective. Both wallets should be owned by the user 🤔

Reply:
@noeliaSD Any thoughts on this?

Reply:
I don't have the full information here but my understanding is.. Here we should display all wallet addresses where I can do transactions. 

@benjthayer could you confirm?

Reply:
> I don't have the full information here but my understanding is.. Here we should display all wallet addresses where I can do transactions.
> 
> @benjthayer could you confirm?

That would be my understanding - if the user has the ability to transact using the account (i.e. not watch only) the account should be able to be used for minting purposes. A Keycard account would just require the user to authenticate the transaction using the relevant Keycard at this point:
https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?type=design&node-id=29384-568047&t=1KMcB8L5M6fBcY58-4